dialogue-test-common/src/main/java/com/palantir/dialogue/AbstractSampleServiceClientTest.java

```diff
@@ -67,8 +63,11 @@ public abstract class AbstractSampleServiceClientTest {
     static final SslConfiguration SSL_CONFIG = SslConfiguration.of(
             Paths.get("src/test/resources/trustStore.jks"), Paths.get("src/test/resources/keyStore.jks"), "keystore");
 
-    @Rule
-    public final MockWebServer server = new MockWebServer();
+    private final MockWebServer server;
+
+    protected AbstractSampleServiceClientTest(MockWebServer server) {
```

Comment:
I didn't see any tests out side this repo that extend this class, so while this is technically an API break I think we're fine